webpack.config.js

```diff
@@ -1,72 +1,76 @@
-var path = require('path');
-const VueLoaderPlugin = require('vue-loader/lib/plugin');
+var path = require("path");
+const VueLoaderPlugin = require("vue-loader/lib/plugin");
 
-module.exports = env => {
-    var baseDir = '';
-    var buildDir = 'app/build/';
-    var buildDirLegacy = 'app/build_legacy/';
-    var entryScript = './src/js/mainScript.js';
-    var outputFilename = 'asterics-grid.bundle.js';
-    var mode = env && env.production ? 'production' : 'development';
+module.exports = (env) => {
+    var baseDir = "";
+    var buildDir = "app/build/";
+    var buildDirLegacy = "app/build_legacy/";
+    var entryScript = "./src/js/mainScript.js";
+    var outputFilename = "asterics-grid.bundle.js";
+    var mode = env && env.production ? "production" : "development";
 
     var scssRule = {
         test: /\.(s*)css$/,
-        use: ['style-loader', 'css-loader']
+        use: ["style-loader", "css-loader"]
     };
 
     var vueRule = {
         test: /\.vue$/,
-        loader: 'vue-loader'
+        loader: "vue-loader"
     };
 
     var babelRule = {
         test: /\.m?js$/,
         use: {
-            loader: 'babel-loader',
+            loader: "babel-loader",
             options: {
                 presets: [
-                    ['@babel/env', {
-                        modules: false,
-                        useBuiltIns: false,
-                        targets: {
-                            browsers: [
-                                '> 1%',
-                                'last 2 versions',
-                                'Firefox ESR',
-                                'ie >= 11'
-                            ],
-                        },
-                    }]
-                ],
-            },
-        },
+                    [
+                        "@babel/env",
+                        {
+                            modules: false,
+                            useBuiltIns: false,
+                            targets: {
+                                browsers: [
+                                    "> 1%",
+                                    "last 2 versions",
+                                    "Firefox ESR",
+                                    "ie >= 11"
+                                ]
+                            }
+                        }
+                    ]
+                ]
+            }
+        }
     };
 
     var resolve = {
         alias: {
             //objectmodel: "../../../node_modules/objectmodel/dist/object-model.js"
-            vue: 'vue/dist/vue.esm.js'
+            vue$: "vue/dist/vue.esm.js",
+            "@": path.resolve("src")
```

Comment:
This makes it so you can reference files easier. `import {constants} from "../util/constants";` -> `import {constants} from "@/js/util/constants";`

Though I believe I wrote it wrong. It should look like this:
```
            "@": path.resolve(__dirname, 'src'),
            "@assets": path.resolve(__dirname, 'app')
```

Reply:
ok, but please keep this PR focussed on Prettier and let's don't add or change other things.